client/libs/core-data/src/lib/sermons/sermons.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable } from '@angular/core';
+
+import { Apollo } from 'apollo-angular';
+import { ApolloQueryResult } from 'apollo-client';
+
+import {
+  sermonQuery,
+  createSermonMutation,
+  updateSermonMutation,
+  deleteSermonMutation
+} from './sermons.graphql';
+import { Sermon } from './sermons.model';
+
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SermonsService {
+
+  constructor(private apollo: Apollo) { }
+
+  all(): Observable<Sermon[]> {
+    return this.apollo.query({
+      query: sermonQuery,
+      fetchPolicy: 'network-only'
+    }).pipe(map((response: ApolloQueryResult<any>) => response.data.sermons))
+  }
+
+  create(sermon: Partial<Sermon>) {
+    delete sermon.id;
+    delete sermon.date;
```

Comment:
Keep this in mind.
We don't always want to delete the sermon date.
What if there is a date being sent through?